llms/mlx_lm/manage.py

```diff
@@ -18,7 +20,7 @@ def ask_for_confirmation(message: str) -> bool:
             return True
         if answer in n:
             return False
-        print(f"Invalid input. Must be one of {all_values}")
+        print(f"Invalid input. Must be one of: yes/no/y/N or empty for no")
```

Comment:
Is it intentional that the `y` lower case but the `N` upper case? Could we make them the same?

Reply:
You are right. Fixed.